thavalon-webapp/src/components/Game.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { GameSocket, InboundMessage } from '../utils/GameSocket';
+
+export function Game(): JSX.Element {
```

Comment:
This function can be deleted if testing is complete, since the lobby page is coming in #65